pkg/controller/common.go

```diff
@@ -1983,19 +1982,22 @@ func (vq *VolumeReplicaRequestsPriorityQueue) DrainQueue() []*ReplicaRequest {
 
 ///Removes replica requests from the priority queue and adds to operation queue.
 func (c *SharedState) tryCreateFailedReplicas(ctx context.Context, requestor operationRequester) error {
```

Comment:
This function always returns `nil`, i.e. it never fails. We should change it so that it returns nothing so the caller doesn't have to bother to check the error return value. This can be done in a separate PR, though.